dipy/workflows/multi_io.py

```diff
@@ -134,6 +134,8 @@ def io_iterator(inputs, out_dir, fnames, output_strategy='absolute',
         Controls the behavior of the IOIterator for output paths.
     mix_names : bool
         Whether or not to append a mix of input names at the beginning.
+    out_keys : list, optional
+        Output parameter names.
```

Comment:
can you add the optional for `mix_names` and `output_strategy` ?  Thanks

Reply:
Done in 7cf0a36. Push-forced.